man/systemd.exec.xml

```diff
@@ -936,7 +936,18 @@
         <para>Note that for each unit making use of this option a PAM session handler process will be maintained as
```

Comment:
"Note that" occurs 12 times as the start of a paragraph in this manpage alone. Not sure what this tells us though 😉.

Reply:
That I like the expression? ;-) I figure I was the one who wrote those paragraphs originally...